portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,186 @@
-#content {
+.center {
+  display: block;
   margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin-right: 25%;
+  width: 50%;
 }
 
-#greeting-container {
+h1 {
+    padding: 20px;
```

Comment:
Could you please alphabetize the declarations for this `h1` and all of the other ones in this file? It makes it much easier to scan through at a glance in the future.

ex. 
```
    color: black;
    font-family: Georgia, 'Times New Roman', Times, serif;
    font-size: 1;
    padding: 20px;
    text-align: center;
```

https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order